examples/mcmc_examples/PyFstat_example_fully_coherent_MCMC_search.py

```diff
@@ -1,8 +1,17 @@
+"""
+MCMC search: Fully coherent F-statistic
+=======================================
+
+Targeted MCMC search for an isolated CW signal using the
+fully coherent F-statistic. This is functionaly equivalent
+to a semicoherent F-statistic search using a single segment.
```

Comment:
```suggestion
Directed MCMC search for an isolated CW signal using the
fully coherent F-statistic.
```

(That comment about coherent <=> Nseg=1 is a bit too randomly generic for this specific file.)